src/frontend/src/components/items/YesNoButton.tsx

```diff
@@ -0,0 +1,18 @@
+import { t } from '@lingui/macro';
+import { Badge } from '@mantine/core';
+
+export function YesNoButton(value: any) {
```

Comment:
I think you forgot the destructuring.

```suggestion
export function YesNoButton({ value: any }) {
```

Reply:
Thanks @wolflu05 - https://github.com/inventree/InvenTree/pull/5711